CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+without having to sacrifice any determinism. By leveraging this feature, Cardano would gain two
+unique abilities:
+
+1. Users would be able to trustlessly build up transactions off-chain without having to coordinate
+   with each other.
+2. Transaction pieces can possibly be unbalanced.
+
+These two properties are exactly the requirements for a simple, expressive, and efficient
+(economically and computationally) intent settlement system. 
+
+While this CIP would introduce some minor duplication in transactions and blocks, the fact that the
+transaction fee is also based on the size of the transaction means all users would be naturally
+incentivized to minimize this duplication as much as possible. The ability to push a significant
+amount of activity off-chain (like in the high frequency trading example) would likely more than
+make up for any remaining duplicate information in each transaction/block. Furthermore, the impact
+this CIP has on the average block size seems smaller than the impact the main alternative proposals
+would have on the average block size.
+
+## Alternatives
+
+### Validation Zones
+
+[Validation Zones](https://github.com/cardano-foundation/CIPs/pull/862) is another intent settlement
+system proposal, but it is unable to properly model all intents since it doesn't allow for
+unbalanced intents (ie, intents without counter-party information). As a result of the model not
+accurately representing all intents, the proposal requires a lot of extra complexity and friction
+with its usage.
+
+The Validation Zones CIP fundamentally models intents as full transactions which means all intents
+must be balanced. Without the ability to specify unbalanced intents, Validation Zones forces users
+to try to guess what actions the counter-party will use when satisfying their intent. In practice,
+this means Alice must use input and output **placeholders** in her intent, and these placeholders
+must also specify the value of the UTxO that will eventually go there; these placeholders are
+necessary to finish balancing the intent transaction. But this creates problems for potential
+counter-parties: 
+
+- Only UTxOs that exactly fit those placeholders can be used
+- The counter-party can't add or remove placeholders after the fact
+
+Imagine if Alice wants to swap 100 ADA for 50 DJED; this says nothing about how the counter-party
+will satisfy her swap. Perhaps Bob (the counter-party) wants to provide the DJED, but it is
+currently broken up across three UTxOs. And perhaps he wants the 100 ADA he receives to be stored in
+two separate outputs. If Alice specifies only one input placeholder and one output placeholder in
+her intent transaction, Bob might be unable to act as her counter-party. If Bob really wanted to
+still trade with Alice, he would have to find a way to merge his inputs and outputs into one input
+and one output. This may not always be possible and Bob may have to give up on being Alice's
```

Comment:
This is actually quite easy to do with validation zones. Bob just needs to spend his three outputs and Alice's Ada output and negative Djed output, and make two new outputs containing the Ada. The only extra effort is to spend Alice's two outputs.

Reply:
See my other comment [here](https://github.com/cardano-foundation/CIPs/pull/873#discussion_r1705441782).